src/modules/auto-oembed.js

```diff
@@ -40,8 +40,12 @@
 	var embedOne = function (ctx, force) {
 		var vPlayer = ctx.find(PLAYER_SEL);
 		var autoLoad = vPlayer.attr('data-autoload');
-		
-		if (!force) {
+
+		var provider = vPlayer.attr('data-oembed-provider');
+		var videoPlayers = ['Vimeo', 'Youtube'];
```

Comment:
I think these should be set inside their own providers

Reply:
Yup, like `provider.requiresVideo()`